assembly/index.ts

```diff
@@ -44,6 +44,8 @@ export declare function assembly_script_address_from_public_key(public_key: stri
 export declare function assembly_script_get_time(): u64;
 @external("massa", "assembly_script_unsafe_random")
 export declare function assembly_script_unsafe_random(): i64;
+@external("massa", "assembly_script_send_message")
+export declare function assembly_script_send_message(target_address: string, target_handler: string,validity_start_period: i64, validity_start_thread: i64, validity_end_period: i64, validity_end_thread: i64, max_gas: u64, gas_price: string, coins: string, payload: string): void;
```

Comment:
```suggestion
export declare function assembly_script_send_message(target_address: string, target_handler: string, validity_start_period: i64, validity_start_thread: i64, validity_end_period: i64, validity_end_thread: i64, max_gas: u64, gas_price: string, coins: string, payload: string): void;
```
Also, why `validity_start_period:`, `validity_start_thread`, `validity_end_period` and `validity_end_thread` are `i64` ? Why reading their name it don't seems they could be negative

Reply:
https://www.assemblyscript.org/types.html

good catch, they should be positive. Also the "thread" should be represented as an i32 in the ABI and as u8 in assemblyscript